app/templates/inbox/meetings/new.hbs

```diff
@@ -0,0 +1,70 @@
+{{!-- TODO: Uncomment this once the app uses `{{page-title}}` on all relevant pages --}}
+{{!-- {{page-title (t "inbox.meetings.new.title")}} --}}
```

Comment:
Using this overrides the title that's set when the app boots. I think page titles are useful though. Could be a nice improvement to add this to all relevant pages in the future?

Reply:
It should be possible to uncomment that once #118 is merged.

Reply:
I've merged #118 :)